<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuestWindow</class>
 <widget class="QDialog" name="GuestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="TabObjects">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>541</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #2c3e50, stop: 0.4 #2c3e50,
                                stop: 0.5 #2c3e50, stop: 1.0 #2c3e50);
    border: 2px solid #7f8c8d;
    border-bottom-color: #7f8c8d; /* same as the pane color */
    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    min-width: 150px;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #2980b9, stop: 0.4 #2980b9,
                                stop: 0.5 #2980b9, stop: 1.0 #ecf0f1);
}

QTabBar::tab:selected {
    border-color: #7f8c8d;
    border-bottom-color: #7f8c8d; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 5px; /* make non-selected tabs look smaller */
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="security_tab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>533</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>532</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <attribute name="title">
     <string>Security</string>
    </attribute>
    <widget class="QLabel" name="SecurityLogo">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Finalcons/padlock.png</pixmap>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_Security">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>631</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="cursorWidth">
      <number>0</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QLabel" name="SecurityBackground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>533</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>guest_backgrounds/guest2-01.png</pixmap>
     </property>
    </widget>
    <zorder>SecurityBackground</zorder>
    <zorder>textEdit_Security</zorder>
    <zorder>SecurityLogo</zorder>
   </widget>
   <widget class="QWidget" name="temperature_tab">
    <attribute name="title">
     <string>Temperature</string>
    </attribute>
    <widget class="QLabel" name="TemperatureLogo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Finalcons/thermometer.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="TemperatureBackground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>533</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>guest_backgrounds/guest3-01.png</pixmap>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_Temperature">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="cursorWidth">
      <number>0</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <zorder>TemperatureBackground</zorder>
    <zorder>TemperatureLogo</zorder>
    <zorder>textEdit_Temperature</zorder>
   </widget>
   <widget class="QWidget" name="Garage_tab">
    <attribute name="title">
     <string>Garage</string>
    </attribute>
    <widget class="QLabel" name="GarageLogo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Finalcons/garage.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="GarageBackground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>533</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>guest_backgrounds/guest5-01.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="GarageState">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>230</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>GarageBackground</zorder>
    <zorder>GarageLogo</zorder>
    <zorder>GarageState</zorder>
   </widget>
   <widget class="QWidget" name="Light_tab">
    <attribute name="title">
     <string>Light</string>
    </attribute>
    <widget class="QLabel" name="LightLogo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Finalcons/light-bulb.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="LightBackground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>533</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>guest_backgrounds/guest1-01.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="LightState">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>230</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>LightBackground</zorder>
    <zorder>LightLogo</zorder>
    <zorder>LightState</zorder>
   </widget>
   <widget class="QWidget" name="Tap_tab">
    <attribute name="title">
     <string>Tap</string>
    </attribute>
    <widget class="QLabel" name="TapLogo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>180</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Finalcons/droplet.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="TapBackground">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>533</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>guest_backgrounds/guest4-01.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="TapState">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>370</y>
       <width>230</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>TapBackground</zorder>
    <zorder>TapLogo</zorder>
    <zorder>TapState</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
